### DISCUSSION TITLE
Banner for Windows installer

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#462] [Development]
Hi @zhuiks @augustin-colesnic,

I switched to a different windows installer framework (electron/windows-installer), because the previously used one (electron-installer-windows) stopped working with GitHub Actions.

As part of that effort I added a "loading gif" to the installer, which is shown to the user during installation on Windows.

It looks like this:
![image](https://user-images.githubusercontent.com/17739895/135755765-3e43d2e0-fa3d-4d14-b43a-d91b1f4888c1.png)

I just quickly threw something together ... it could technically be animated (gif) and it could probably look nicer. So if any of you guys finds some time to create a nice Gif ... go for it!

**Comment by zhuiks** [comment]
The opening scroll animation comes to mind...
